src/main/java/ch/njol/skript/effects/EffPlaySound.java

```diff
@@ -236,10 +242,18 @@ static <T, E> void play(
 					key = enumSound.getKey();
 				} catch (IllegalArgumentException alternative) {
 					sound = sound.toLowerCase(Locale.ENGLISH);
-					if (!KEY_PATTERN.matcher(sound).matches())
+					Matcher keyMatcher = KEY_PATTERN.matcher(sound);
+					if (!keyMatcher.matches())
 						continue;
 					try {
-						key = NamespacedKey.fromString(sound);
+						String namespace = keyMatcher.group(1);
+						String keyValue = keyMatcher.group(2);
+						if (namespace == null) {
+							key = NamespacedKey.minecraft(keyValue);
+						} else {
+							namespace = namespace.substring(0, namespace.length() - 1);
```

Comment:
Couldn't we just not capture the `:`? It doesn't really matter I suppose

Reply:
Yeah it'd make it a little more awkward when we do use fromString. Either way, one of them needs a little more processing.